lib/oauth.js

```diff
@@ -155,6 +165,15 @@ module.exports = new Model({
     ].join('');
   },
 
+  _bearerTokenIsExpired: function() {
```

Comment:
Do we need a new method to check if token is expired?[ One already existed](https://github.com/zooniverse/panoptes-javascript-client/blob/9a1cab1227d88924c810bee3cf3b39b558bc9d08/lib/oauth.js#L307).

Reply:
https://github.com/zooniverse/shakespeares_world/issues/366#issuecomment-362605881

Reply:
We need a method that works. I'd prefer if the OAuth class has the same API as Auth, so that common code can be consolidated as per the recommendations in #76.

Reply:
Makes sense, sorry I forgot you already commented on that.